en/modules/ROOT/nav.adoc

```diff
@@ -19,6 +19,7 @@
 *** xref:customize:texts.adoc[Texts]
 *** xref:customize:users_registration_mode.adoc[User registration mode]
 *** xref:customize:views.adoc[Views]
+*** xref:customize:logic.adoc[Logic]
```

Comment:
This should be ordered alphabetically, so it should be before the `Menu` entry at line 16, can you move it please? Thanks